markdown/util.py

```diff
@@ -85,19 +85,7 @@
 # Extensions should use "markdown.util.etree" instead of "etree" (or do `from
 # markdown.util import etree`).  Do not import it by yourself.
```

Comment:
These comments should be rewritten as they are no longer correct. Instead, we would now be importing `etree` here for backward compatibility with old third party extensions which used to rely on our custom import.

However, as that custom import is no longer necessary, the `markdown` library itself should not use this import at `markdown.util` but simply import `tree` directly. Perhaps we could even raise a deprecation warning if third party code uses this import.